### ISSUE TITLE
The docker postgres database container stops running after a time.

### ISSUE THREAD

**Issue by jlinoff** [jlinoff/grafmdev#10]
The docker postgres database container stops running after a time. It is not clear how long it runs before stopping but it is more than an hour.

I suspect that the problem is related to the `/tmp` or `/var/lib/postgresql/data` volume mounts.

**Comment by jlinoff** [comment]
My first test will replace `/tmp` with `/mnt` and test for 12h.

**Comment by jlinoff** [comment]
How i am checking elapsed time for `demo01pg` using GNU date (gdate):
```bash
$ gdate -u -d @$(echo "$(gdate -u +%s) - $(gdate --date $(docker inspect -f '{{ .Created }}' demo01pg) +%s)" | bc) +'%T'
02:53:18
```

**Comment by jlinoff** [comment]
After more than 5 hours things the test runs are working.

**Comment by jlinoff** [comment]
Fixed in v0.6.0.
